CONTRIBUTING.md

```diff
@@ -59,11 +59,9 @@ The best way to create a minimal reproduction is gradually removing code and dep
 
 ## Contributing Changes
 
-Project maintainers will merge changes that improve the product significantly and broadly align with the [.NET Roadmap](https://github.com/dotnet/core/blob/master/roadmap.md).
+Project maintainers will merge changes that improve the product significantly. Changes that break compatibility or take on other risk will require community feedback (in an issue) and will typlically be rejected.
```

Comment:
I don't think we can make this claim. Almost anything can break compatibility with something somewhere, we do take such changes, and we don't always have an issue discussing it.

Reply:
Clearly, we don't want to discourage breaks. Do you have alternate wording to suggest?

Reply:
Don't have the second sentence at all?

Reply:
```suggestion
Project maintainers will merge changes that improve the product significantly.
```